src/stacks/nestedAccount.ts

```diff
@@ -25,20 +25,51 @@ export class NestedAccountAlarmsStack extends cfn.NestedStack {
   }
 
   private setupAccountAlarm(metricName: string): void {
-    const autoResolve = config.configAutoResolve(config.ConfigDefaultType.Account, metricName);
+    const conf = config.configGetAllDefaults(config.ConfigDefaultType.Account)?.[metricName];
+    if (!conf) {
+      return;
+    }
+
+    const isEnabled = conf.enabled;
```

Comment:
This const is redundant.

Reply:
That is actually bug, it should check if the value is false so undefined is marked as enabled